proto/backend.proto

```diff
@@ -131,7 +131,28 @@ message DataResponse {
   // Frame has its own meta, warnings, and repeats refId
   repeated bytes frames = 1;
   string error = 2;
-  bytes jsonMeta = 3; // Warning: Current ignored by frontend. Would be for metadata about the query.
+  bytes jsonMeta = 3; // Warning: Currently ignored by frontend. Would be for metadata about the query.
+  ErrorDetails errorDetails = 4;
+}
+
+message ErrorDetails {
+  Status status = 1;
+
+  enum Status {
+    UNKNOWN = 0;
+    UNAUTHORIZED = 1;
+    FORBIDDEN = 2;
+    NOT_FOUND = 3;
+    TOO_MANY_REQUESTS = 4;
+    BAD_REQUEST = 5;
+    VALIDATION_FAILED = 6;
+    INTERNAL = 7;
+    NOT_IMPLEMENTED = 8;
+    TIMEOUT = 9;
+    BAD_GATEWAY = 10;
+  }
+
+  string message = 2;
```

Comment:
this is a bit odd that the error (line 133) exists, and there is an error message